### DISCUSSION TITLE
Can't download dav5x form F-Droid

### DISCUSSION THREAD

**Discussion by ghost** [community/community#128018] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I used to sync my Fruux-calender via DAV5X to my calender-app on my smartphone.
That worked well for several years, until a week ago. I removed DAV5X form my smartphone to reinstall it again,
but now, I can't install it anymore. I get the message in attach.
I have an old phone: Huawei P8 Lite with Android 8, so it should work...
I'm new, so I hope this is the right place for this question.
Can someone help me out? Many thanks! 

![Screenshot_20240531-162533](https://github.com/community/community/assets/172280302/e144c3c7-aaea-4f5e-b638-0581c1a2ce3d)

**Comment by Rishi-Sudhakar** [comment]
The apk seems to be broken. Try Reinstalling, or try older versions of the apk. They seem to have changed a number of things - (https://f-droid.org/en/2024/05/23/twif.html) .

**Reply by IhfazAlam** [reply]
Hello

**Reply by IhfazAlam** [reply]
Hello, Rishi-Sudhakar

**Reply by IhfazAlam** [reply]
Thanks for following me

**Reply by IhfazAlam** [reply]
Bye

**Comment by IhfazAlam** [comment]
Hello I have problem

**Reply by IhfazAlam** [reply]
Can you help me

**Reply by IhfazAlam** [reply]
Plz reply me quickly

**Comment by ghost** [comment]
Hello, thank you for your help!
There are three versions that I can select on my phone:
4.3.12.1-ose (27/12/23), 4.3.12-ose (25/12/23), and 4.3.11-ose (22/11/23).
For each of them I receive the same kind of error-message:

![Screenshot_20240610-202640](https://github.com/community/community/assets/172280302/08c9afc5-0091-4f2a-a57b-67009b2cfea9)

![Screenshot_20240610-202617](https://github.com/community/community/assets/172280302/a9e707b2-405d-48ac-bab4-61055e61dbdf)
